src/utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * Simple utility function used to find an item in a set with a predicate
+ * function.  Analagous to Array.find(), without requiring converting the set
+ * an Array.
+ */
+export function findInSet<T>(set: Set<T>, predicate: (item: T) => boolean): T|
```

Comment:
Could be even more generic with `Iterable<T>` I think?

Reply:
Neat!  Doing that and renamed to `find`